conf/application.conf

```
@@ -54,6 +63,12 @@ urls {
   bas-gateway-base-url = ${?platform.frontend.host}
   bas-gateway-sign-in = ${urls.bas-gateway-base-url}"/bas-gateway/sign-in"
   bas-gateway-sign-out = ${urls.bas-gateway-base-url}"/bas-gateway/sign-out-without-state"
+
+  asa-fe-base-url = "http://localhost:9437"
+  asa-fe-dashboard-url = ${urls.asa-fe-base-url}"/agent-services-account/home"
+
+  taxAndSchemeManagementToSelfServeAssignmentOfAsaEnrolment = ${urls.this-frontend}"/taxAndSchemeManagementToSelfServeAssignmentOfAsaEnrolment" //TODO, ask Geoff
```

From usage in agent-services-account:
```suggestion
    user-management-base-url = "http://localhost:1111"
    taxAndSchemeManagementToSelfServeAssignmentOfAsaEnrolment = ${urls.user-management-base-url}"/tax-and-scheme-management/users?origin=Agent"
```